eng/Directory.Build.Data.targets

```diff
@@ -13,20 +13,21 @@
   <PropertyGroup>
     <DefaultReferenceTargets>AzSdk.reference.targets</DefaultReferenceTargets>
     <IsTestProject Condition="'$(IsTestProject)' == '' and $(MSBuildProjectName.EndsWith('.Tests'))">true</IsTestProject>
-    <EnableClientSdkAnalyzers Condition="'$(EnableClientSdkAnalyzers)' == '' and '$(IsClientLibrary)' == 'true' and '$(IsTestProject)' != 'true'">true</EnableClientSdkAnalyzers>
+    <IsPerformanceTestProject Condition="'$(IsPerformanceTestProject)' == '' and $(MSBuildProjectName.EndsWith('.Performance'))">true</IsPerformanceTestProject>
```

Comment:
Maybe we should consider `.PerformanceTests` or `.Performance.Tests` to be more clear?   This would also potentially give you the designation of `IsTestProject` to avoid overloading logic except where you need to differentiate specifically between tests and performance tests.

Reply:
I feel the name is long enough as it is, causes long path issued for me already.